sql/ai.control

```diff
@@ -1,6 +1,7 @@
 # pgai extension
 comment='helper functions for ai workflows'
-default_version='0.3.0'
+default_version='0.4.0'
 requires='vector,plpython3u'
 relocatable=false
+schema=ai
```

Comment:
Yeah I think this makes sense. Let's double check with @avthars as well since this changes all the function names.

Reply:
I think this needs a migration to move existing function from prior versions to the new schema.

Reply:
okay. we're good here.